tensorwaves/physics/helicityformalism/kinematics.py

```diff
@@ -12,7 +12,14 @@
 :class:`~SubSystem`.
 """
 import logging
-from typing import List, Optional, Union
+from collections.abc import Hashable
```

Comment:
Small thought: since [`typing.Hashable`](https://docs.python.org/3/library/typing.html#typing.Hashable) is not the same as [`abc.Hashable`](https://docs.python.org/3/library/collections.abc.html#collections.abc.Hashable) (this is the case for many typing vs abstract base metaclasses), I tend to use `from collections import abc` and use `abc.WhateverClass`, then `from typing import WhateverClass` for the actual typing, so that there is no conflict between the two and the intension is clear from the namespace. It doesn't really matter here, because we only use `abc.Hashable` to enforce `__hash__` definition, so consider this a general thought.